jcl/src/java.base/share/classes/jdk/crac/Core.java

```diff
@@ -41,7 +41,12 @@ public class Core {
 	 */
 	public static Context<Resource> getGlobalContext() {
 		if ((globalContext == null) && InternalCRIUSupport.isCRaCSupportEnabled()) {
-			globalContext = new CRIUSupportContext<>();
+			try {
+				globalContext = new CRIUSupportContext<>();
+			} catch (IllegalArgumentException e) {
+				System.err.println("Invalid checkpoint directory supplied: " + InternalCRIUSupport.getCRaCCheckpointToDir());
```

Comment:
I thought `//$NON-NLS-1$` doesn't have any impact anymore. For consistency in its usage, it is missing over here.
```suggestion
				System.err.println("Invalid checkpoint directory supplied: " + InternalCRIUSupport.getCRaCCheckpointToDir()); //$NON-NLS-1$
```

Reply:
Im not sure what the rules are on this, do you know @pshipton ?